test/plugin/localizedFormat.test.js

```diff
@@ -96,6 +96,16 @@ it('Uses the localized lowercase formats if defined', () => {
   ['l', 'll', 'lll', 'llll'].forEach(option => expect(znDate.format(option)).toBe(znDate.format(znCn.formats[option])))
 })
 
+it('Uses fallback to xx if xx-yy not available', () => {
+  expect(dayjs('2019-02-01').locale('es-yy').format('L'))
```

Comment:
it will be more clear to use `format('MMMM')` to test if the locale is correct and not the default English